### PR TITLE
cli: avoid default value confusion for flag help text

### LINKED ISSUE #6066: Wrong defaults in log flag help
With [PR 5979](https://github.com/cockroachdb/cockroach/pull/5979), logs are no longer written to stderr by default. 

However, the default values for `--alsologtostderr` and `--logtostderr` in command `help` seem out of sync with this change:

```
Flags:
      --alsologtostderr value[=INFO]   logs at or above this threshold go to stderr
      --log-backtrace-at value         when logging hits line file:N, emit a stack trace (default :0)
      --log-dir value                  if non-empty, write log files in this directory
      --logtostderr value[=true]       log to standard error instead of files
      --no-color value                 disable standard error log colorization
      --verbosity value                log level for V logs
      --vmodule value                  comma-separated list of pattern=N settings for file-filtered logging
```

If logs are no longer written to stderr by default, I would expect `--alsologtostderr value[=NONE]` and `--logtostderr value[=false]`.

@petermattis, @tschottdorf

### PR DESCRIPTION
**Pull request by petermattis**
Fixes #6066.

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/6070)

### CODE REVIEW

File: cli/cli_test.go [hunk 1]
```diff
@@ -754,10 +754,10 @@ Available Commands:
   debug       debugging commands
 
 Flags:
-      --alsologtostderr value[=INFO]   logs at or above this threshold go to stderr
+      --alsologtostderr value[=INFO]   logs at or above this threshold go to stderr (default NONE)
```
Comment (tschottdorf) [inline cli/cli_test.go]: nit: `--alsologtostderr=NONE` sounds like it should work.

**Comment by tschottdorf** [comment]
LGTM

File: cli/cli_test.go (hunk 1 above)
Comment (petermattis) [inline cli/cli_test.go]: Fair point. Fixed.

**Comment by tschottdorf** [comment]
LGTM (again).

**Pull request merged**